mod/deckorate_graphql/lib/graph_q_l/types/deckorate_fields.rb

```diff
@@ -0,0 +1,53 @@
+module GraphQL
+  module Types
+    module DeckorateFields
+
+      def lookup_field fieldname, type, codename = nil, is_card = false
+        codename ||= fieldname
+        plural_fieldname = fieldname.to_s.to_name.vary(:plural).to_sym
+        is_card ||= is_card
+
+        plural_field plural_fieldname, codename, type
+
+        define_method plural_fieldname do |limit: 10, offset: 0, **filter|
+          lookup_search codename, limit, offset, is_card, filter
+        end
+      end
+
+      def card_field fieldname, type, codename = nil
+        codename ||= fieldname
+        singular_field fieldname, type
+        define_method(fieldname) { |**mark| ok_card codename, **mark }
+      end
+
+      def cardtype_field fieldname, type, codename = nil, is_card = false
+        codename ||= fieldname
+        plural_fieldname = fieldname.to_s.to_name.vary(:plural).to_sym
+        plural_field plural_fieldname, codename, type
+
+        define_method plural_fieldname do |limit: 10, offset: 0, **filter|
+          card_search codename, limit, offset, is_card, filter
+        end
+      end
```

Comment:
did we decide not to move card_field and cardtype_field to the more general graphql module?